templates/membership/join.html.twig

```diff
@@ -163,6 +163,9 @@
                         {% endif %}
 
                         <div class="register__form--full">
+                            {% if formIsValid is same as(false) %}
```

Comment:
What about using form.vars.valid instead ?

Reply:
I didn't see this var. I think it's better than doing this trix ^^,. Thx